app/models/manageiq/providers/ansible_tower/automation_manager/refresh_worker/runner.rb

```diff
@@ -1,2 +1,3 @@
 class ManageIQ::Providers::AnsibleTower::AutomationManager::RefreshWorker::Runner < ManageIQ::Providers::BaseManager::RefreshWorker::Runner
+  # FIXME why do we even need this class?
```

Comment:
Right, this is the physical process that MIQ monitors for the AnsibleTower Automation Manager refresh worker.  Whereas the `AnsibleTower::AutomationManager::RefreshWorker` is the `ApplicationRecord` that is stored in the `miq_workers` table for capturing the details of this process.